README.md

```diff
@@ -27,52 +27,52 @@ npm i --save @hubspot/calling-extensions-sdk
 ```shell
 yarn add @hubspot/calling-extensions-sdk
 ```
-## Run the Demo Call Widget
+## Run the Demos
 
-We have installed the SDK on a demo call widget as an example.
+We have installed the SDK on two demo apps to serve as examples:
+- The [demo-minimal-js](https://github.com/HubSpot/calling-extensions-sdk/tree/project-demo-v1/demos/demo-minimal-js) features a minimal implementation of the SDK using JavaScript, HTML, and CSS. View how the SDK is instantiated on [index.js, line 62](https://github.com/HubSpot/calling-extensions-sdk/blob/project-demo-v1/demos/demo-minimal-js/index.js#L62).
```

Comment:
We could probably omit the line number here since this could change over time?

Reply:
Great callout. Done!